### PR TITLE
swapped function names for percentage of reoccuring

### PR DESCRIPTION
**Pull request by emanuelef**
As per issue #724 (https://github.com/blue-yonder/tsfresh/issues/724).

percentage_of_reoccurring_datapoints_to_all_datapoints is actually percentage_of_reoccurring_values_to_all_values implementation and vice versa .

### CODE REVIEW

File: tsfresh/feature_extraction/feature_calculators.py [hunk 1]
```diff
@@ -891,7 +891,7 @@ def percentage_of_reoccurring_datapoints_to_all_datapoints(x):
 
 @set_property("fctype", "simple")
 @set_property("input", "pd.Series")
-def percentage_of_reoccurring_values_to_all_values(x):
+def percentage_of_reoccurring_datapoints_to_all_datapoints(x):
     """
     Returns the ratio of unique values, that are present in the time series
     more than once.
```
Comment (nils-braun) [inline tsfresh/feature_extraction/feature_calculators.py:897]: Could you also fix the comment further down (And also in the other function?). It still references the old function name

File: tsfresh/feature_extraction/feature_calculators.py [hunk 2]
```diff
@@ -863,7 +863,7 @@ def first_location_of_minimum(x):
 
 
 @set_property("fctype", "simple")
-def percentage_of_reoccurring_datapoints_to_all_datapoints(x):
+def percentage_of_reoccurring_values_to_all_values(x):
     """
     Returns the percentage of unique values, that are present in the time series
```
Comment (MaxBenChrist) [inline tsfresh/feature_extraction/feature_calculators.py:868]: I think the function descriptions are misleading. I would change it to

```suggestion
    Returns the percentage of values in the time series, that are present 
```

File: tsfresh/feature_extraction/feature_calculators.py (hunk 1 above)
Comment (MaxBenChrist) [inline tsfresh/feature_extraction/feature_calculators.py:897]:
```suggestion
    Returns the fraction of non-unique data points. Non-unique means that they are contained another time in the time series again.
```

**Comment by MaxBenChrist** [comment]
While looking at this feature, I am not sure if it wouldn't make sense to add some rounding before you calculate the number of reocurring values to deal with floating point imprecision?

File: tsfresh/feature_extraction/feature_calculators.py (hunk 1 above)
Comment (emanuelef) [reply tsfresh/feature_extraction/feature_calculators.py:897]: Missed that! Fixed now

**Comment by emanuelef** [comment]
Made some changes according to suggestions, I used "percentage" for both.

**Comment by emanuelef** [comment]
> would be great if you could polish the docstrings a little bit

Sure, what should I change ?

**Comment by nils-braun** [comment]
I think it is fine as it is, thanks for fixing!

**Pull request merged**